server/src/routers/generic.router.ts

```diff
@@ -1,10 +1,13 @@
 import express, { Request, Response } from "express";
 const router = express.Router();
+import authenticateToken from "../middlewares/authentication.middleware";
 import genericController from "../controllers/generic.controller";
+import authController from "../controllers/auth.controller";
+
+router.post("/api/auth/google", authenticateToken, authController.login);
```

Comment:
I think it's better to move auth endpoints to a different router file

Reply:
Okay, would make changes in the next pr.